week-2-practise/level-2/6-map.js

```diff
@@ -2,13 +2,21 @@
 // 1. Write a function that formats the numbers into a string with the percentage symbol appended e.g. "10%"
 // 2. Declare a new array, `percentagesFormatted`, containing
 //    - each item in `percentages` formatted by your function
-
+function percentage(number){
+  return number + "%"
+}
 var percentages = [1, 23, 92, 18];
-
+var percentagesFormatted = percentages.map(percentage)
```

Comment:
With .map() you need to do more than just name a function in the brackets - you must first choose a name that you want to call the items in the array (example: 'item' or in the following example 'num), and you must use this name to refer to each item and tell the program what you want to do to each item.
Example:
```
var doubles = numbers.map(function(num) {
  return num * 2;
});
```
In this example `num` is a name the programmer has created to refer to each item in the array. So first, you pass that item to the function as a parameter, and second you apply an action to that parameter (example: multiply it by 2).

We'll go over this in class on Sunday :)

Reply:
@LucyMac thanks I tried to add a name to the first parenthesis, but there is always an error message